CHANGELOG.md

```diff
@@ -46,6 +48,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   - Update remote message code to handle updated CMA remote message flags
   - Update example SIPS workflows to utilize Parameterized CMA configuration
 
+- **CUMULUS-1449**
+  - `queue-pdrs` config changes:
+    - `parsePdrMessageTemplateUri` replaced with `parsePdrWorkflow`, which is the workflow name (i.e. top-level name in `config.yml`).
+    - `internalBucket` and `stackName` configs now required to look up configuration from the deployment. Brings the task config in line with that of `queue-granules`.
+  - `queue-granules` config change: `ingestGranuleMessageTemplateUri` replaced with `ingestGranuleWorkflow`, which is the workflow name.
+  - Cumulus now uses a universal workflow template when starting workflow that contains general information specific to the deployment, but not specific to the workflow. Workflow Task configs must be defined using AWS step function parameters. See following entry.
```

Comment:
```suggestion
  - Cumulus now uses a universal workflow template when starting workflow that contains general information specific to the deployment, but not specific to the workflow. Workflow task configs must be defined using AWS step function parameters. See following entry.
```

Reply:
Should this and the note below just be moved up into the "BREAKING CHANGES" section for visibility?